frontend/src/App.js

```diff
@@ -739,138 +779,186 @@ function App() {
                   * links) */}
                 <Navbar bg="light" expand="lg">
                     <Navbar.Brand>
-                        <p className="h1"> <img src={logo} alt="Scivision logo" /> Scivision</p>
+                        <Nav.Link to="home" as={NavLink}>
+                            <img src={biglogo} width="38%" height="auto" alt="Scivision" />
+                        </Nav.Link>
                     </Navbar.Brand>
                 </Navbar>
 
-                <div className="row px-4 mt-2">
-                    {/* Navigation sidebar */}
-                    <Nav className="col-auto d-block sidebar">
-                        <Nav.Item>
-                            <Link to="">About</Link>
-                        </Nav.Item>
-                        <p />
-
-                        <Nav.Item>
-                            <LoginStatusLink gh_logged_in={gh_logged_in}
-                                             set_gh_logged_in={set_gh_logged_in} />
-                        </Nav.Item>
-                        <p />
-
-                        <Nav.Item>
-                            <Link to="datasource-grid">Datasources</Link>
-                        </Nav.Item>
-
-                        <Nav.Item>
-                            <Link to="model-grid">Models</Link>
-                        </Nav.Item>
-
-                        <p />
+                {/* Navigation bar */}
+                <Navbar bg="light" expand="lg">
+                    <Nav className="me-auto">
+                        <Nav.Link to="" as={NavLink}>
+                            Home
+                        </Nav.Link>
+
+                        <Nav.Link to="about" as={NavLink}>
+                            About
+                        </Nav.Link>
+
+                        <Nav.Link to="scivisionpy" as={NavLink}>
+                            Scivision.Py
```

Comment:
This was going to be a link to a general page about using it in python, with a code snippet and some doc links.  (Perhaps it could even have a jupyterlite repl like the numpy homepage.)

The message I wanted to get across with the link was really that a Scivision python library exists, which might not be obvious if you've just arrived - I'd worry API might cause people to think it's a web API, and also sounds a bit more 'technical'.

Reply:
Right - I got your point. In that case `Scivision.Py` sounds ok.